app/jobs/maestrano/connector/rails/concerns/synchronization_job.rb

```diff
@@ -13,13 +13,23 @@ def enqueued?(organization_id)
     def find_job(organization_id)
       queue = Sidekiq::Queue.new(:default)
       queue.find do |job|
-        organization_id == job.item['args'][0]['arguments'].first
```

Comment:
```ruby
organization_id == job.item['args'][0]['arguments'].first rescue false
```
?

Reply:
it's what I did initially but rubocop does not like inline rescue